packages/hardhat-zksync-verify/src/utils.ts

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import { VerificationStatusResponse } from './zkscan/VerificationStatusResponse';
+import { checkVerificationStatus } from './zkscan/ZkScanService';
+import { ZkSyncVerifyPluginError } from './zksync-verify-plugin-error';
+
+export function handleAxiosError(error: any): never {
+    if (axios.isAxiosError(error)) {
+        throw new Error(
+            `Axios error (code: ${error.code}) during the contract verification request\n Reason: ${error.response?.data}`
+        );
+    } else {
+        throw new ZkSyncVerifyPluginError(`Failed to send contract verification request\n Reason: ${error}`);
+    }
+}
+
+export function delay(ms: number) {
```

Comment:
Maybe should be under a generic utils for all plugins? Feel free to ignore if just creates more confusion

Reply:
Plugins are separate packages, I don't think they can share files like that.

Reply:
Yes, each plugin represents a mini project on its own, so it is best to keep their utils files separated.